kafka-connect-cloud-common/src/main/scala/io/lenses/streamreactor/connect/cloud/common/formats/writer/JsonFormatWriter.scala

```diff
@@ -101,4 +95,10 @@ object JsonFormatWriter {
     Map("schemas.enable" -> "false", JsonConverterConfig.DECIMAL_FORMAT_CONFIG -> DecimalFormat.NUMERIC.name()).asJava,
     false,
   )
+
+  def getIfSupported: CompressionCodec => CompressionCodec = {
```

Comment:
can this be something more descriptive, perhaps `getCompressionCodecIfSupported`?